packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -34,12 +34,6 @@ const args = arg(
     '--no-mini-proxy-default-ca': Boolean,
     // Enable debug logs in the bundled Mini-Proxy server
     '--mini-proxy-debug': Boolean,
-    // Passes the same flag to Jest to only run tests related to changed files
```

Comment:
I think it's still nice to have support for these things locally

Reply:
+1, I created a separate PR